src/run/handlers/server.ts

```diff
@@ -48,6 +48,9 @@ const disableFaultyTransferEncodingHandling = (res: ComputeJsOutgoingMessage) =>
 // TODO: remove once https://github.com/netlify/serverless-functions-api/pull/219
 // is released and public types are updated
 interface FutureContext extends Context {
+  flags: {
```

Comment:
This is not meant to be a customer-facing property, so we never added it to the `Context` types. I've added it here for now, but we should consider having a `InternalContext` or similar that extends the main context with any internal properties.

Reply:
Makes sense. Might be worth adding the reason for putting this here to the comment above, until `InternalContext` is implemented